Cargo.toml

```
@@ -100,6 +102,7 @@ all-builtins = [
   "json-builtins",
   "semver-builtins",
   "sprintf-builtins",
+  "units-builtins"
```

having a trailing comma should help unnecessarily having to modify this line later

```suggestion
  "units-builtins",
```